### PR TITLE
More user friendly fixture errors

### PR DESCRIPTION
**Pull request by wpride**
re: http://manage.dimagi.com/default.asp?226894

before
screen shot 2016-05-13 at 3 59 16 pm

after
screen shot 2016-05-13 at 3 59 05 pm

### CODE REVIEW

File: touchforms/backend/touchcare.py [hunk 1]
```diff
@@ -179,7 +179,13 @@ def _get_fixture(self, user_id, fixture_id):
         try:
             results = q(query_url)
         except (HTTPError, URLError), e:
-            raise TouchFormsNotFound('Unable to fetch fixture at %s: %s' % (query_url, str(e)))
+            fixture_name = query_url[query_url.rfind('/') + 1:]
+            if "user-group" in fixture_name:
+                raise TouchFormsNotFound('This form requires that the user be in a case sharing group but one could not be found.')
+            elif "commtrack:locations" in fixture_name:
+                raise TouchFormsNotFound('This form requires that the user be assigned to a location but one could not be found.')
```
Comment (dimagimon) [inline touchforms/backend/touchcare.py]: E501 line too long (130 > 115 characters)

File: touchforms/backend/touchcare.py [hunk 2]
```diff
@@ -179,7 +179,13 @@ def _get_fixture(self, user_id, fixture_id):
...
+            else:
+                raise TouchFormsNotFound('Unable to fetch fixture %s. Ensure the logged in user has access to this fixture.' % fixture_name)
```
Comment (dimagimon) [inline touchforms/backend/touchcare.py]: E501 line too long (140 > 115 characters)

File: touchforms/backend/touchcare.py [hunk 3]
```diff
@@ -179,7 +179,13 @@ def _get_fixture(self, user_id, fixture_id):
         try:
             results = q(query_url)
         except (HTTPError, URLError), e:
-            raise TouchFormsNotFound('Unable to fetch fixture at %s: %s' % (query_url, str(e)))
+            fixture_name = query_url[query_url.rfind('/') + 1:]
+            if "user-group" in fixture_name:
+                raise TouchFormsNotFound('This form requires that the user be in a case sharing group but one could not be found.')
```
Comment (dimagimon) [inline touchforms/backend/touchcare.py]: E501 line too long (131 > 115 characters)

**Comment by benrudolph** [comment]
sweet this is awesome

**Comment by benrudolph** [comment]
👍 on ✅

**Pull request merged**